cert-tools/launcher/tests/test_stacker.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+
+from launcher.configuration import CheckBoxConfiguration
+
+
+def create_config_file(tmp_path, filename, content):
+    """
+    Helper function to create config files for testing.
+    """
+    path = tmp_path / filename
+    stacker = CheckBoxConfiguration()
+    stacker.read_string(content)
+    stacker.write_to_file(path)
+    return path
+
+
+@pytest.fixture
+def config_files(tmp_path):
+    """
+    Pytest fixture that creates multiple configuration files for testing.
+    """
+    configs = {
```

Comment:
small note about these tests in general. They are fine but if you include the config name in the values (instead of value1, value2 to v1_config1, v2_config1), the overwriting/stacking is probably easier to follow below